docs/sql-data-sources-hive-tables.md

```diff
@@ -148,6 +148,9 @@ The following options can be used to configure the version of Hive that is used
         <li><code>maven</code></li>
         Use Hive jars of specified version downloaded from Maven repositories. This configuration
         is not generally recommended for production deployments.
+        <li><code>path</code></li>
```

Comment:
nit: also need to update above: "three options" -> "four options".

Reply:
Thanks!